internal/proxy/proxy.go

```diff
@@ -113,21 +113,29 @@ func (p *proxy) oauthProxyHandler(w http.ResponseWriter, r *http.Request) {
 
 	if p.allowedOrg != "" {
 		if orgs.LoggedInto(p.allowedOrg) {
-			p.logger.Info("organization belonging user", zap.String("allowedOrganization", p.allowedOrg), zap.Array("organizations", orgs))
+			p.logger.Info("organization belonging user", zap.String("allowedOrganization", p.allowedOrg), zap.Array("organizations", orgs), zap.String("user", u.Login))
 			w.Header().Set("Content-Type", "application/json")
-			io.Copy(w, userInfoResp.Body)
+			b, err := json.Marshal(u)
+			if err != nil {
+				msg := "failed to marshal body"
+				p.logger.Error(msg, zap.Error(err))
+				w.WriteHeader(http.StatusInternalServerError)
```

Comment:
You can use `http.Error` (https://golang.org/pkg/net/http/#Error).

Reply:
Thanks, solved in https://github.com/KeisukeYamashita/spinnaker-github-proxy/pull/7/commits/ec14660a910952b2e5f3ac86105005a91c171509